readthedocs/builds/constants.py

```diff
@@ -33,11 +33,13 @@
 
 BRANCH = 'branch'
 TAG = 'tag'
+PULL_REQUEST = 'pull_request'
```

Comment:
Shouldn't this be called `external` to keep consistency with the rest of the code (internal/external)?

Reply:
Thanks for the review i'll update everything to `internal` and `external` :)